theia-swift-docker/latest.package.json

```diff
@@ -12,24 +12,21 @@
     },
     "dependencies": {
```

Comment:
I wonder how swift image is different from basic Theia. Does it have any special VS Code extensions?

Reply:
From what I understand, it only adds the [sourcekit](https://github.com/apple/sourcekit-lsp) to the Dockerfile:
https://github.com/theia-ide/theia-apps/pull/312/files#diff-d6530c6780572e7d3185476c147af80fL54

Reply:
ok, so it is consumed a bit differently